apps/emails-and-messages/src/modules/sendgrid/configuration/sendgrid-configuration.router.ts

```diff
@@ -1,20 +1,27 @@
 import { logger as pinoLogger } from "../../../lib/logger";
 import {
-  sendgridCreateConfigurationSchema,
-  sendgridDeleteConfigurationInputSchema,
-  sendgridGetConfigurationInputSchema,
+  sendgridConfigurationIdInputSchema,
+  sendgridCreateConfigurationInputSchema,
   sendgridGetConfigurationsInputSchema,
   sendgridGetEventConfigurationInputSchema,
+  sendgridUpdateApiConnectionSchema,
+  sendgridUpdateBasicInformationSchema,
+  sendgridUpdateChannelsSchema,
   sendgridUpdateEventConfigurationInputSchema,
-  sendgridUpdateOrCreateConfigurationSchema,
+  sendgridUpdateEventSchema,
+  sendgridUpdateSenderSchema,
```

Comment:
I'd wrap them in some object - I added comment about that in previous pr

Reply:
Don't quite see the benefit of such wrapping - array of schemas will be moved from this import to dict in another file. Is there an additional benefit I've missed?